scss/_patterns_notifications.scss

```
@@ -101,6 +101,7 @@ $notification-text-margin-bottom: $spv--large - $spv-nudge;
       background-repeat: no-repeat;
       background-size: unset;
       border: 0;
+      min-height: $spv--large * 2;
```

@mtruj013 That kinda works, but we don't want to hardcode "random" values like that.

The height that we need is based on icon size plus button padding + button border.

Icon size you have as `$default-icon-size`; The button padding/border thing is bit harder to dig out, but it seems to be `$sp-nudge` value (on top and bottom), so I think what we need is:

```suggestion
      // set the height of the button to be size of an icon + padding on top and bottom
      height: calc(2 * $sp-nudge + $default-icon-size);
```

Also use `height` we don't want this button to change height based on content, so we don't need min-height.
And use `calc` to make sure the math works even if units inside the variables don't match.

---

BTW, I'm not 100% sure this code will work, it's just a suggestion how I think it could look like.